src/main/groovy/life/qbic/portal/sampletracking/web/presenters/SampleListPresenter.groovy

```diff
@@ -26,4 +34,32 @@ class SampleListPresenter implements SampleTrackingUpdateOutput{
     void updateFinished() {
         viewModel.samples.clear()
     }
+
+    @Override
+    def updateAvailableLocations(List<Location> locations) {
```

Comment:
then we should get rid of it in the interface?

Reply:
could do that yes. see reason like before. I implemented it regardless :D

Reply:
delete pls

Reply:
separate PR

Reply:
this is needed in the controls so not getting rid of it but isolating it in another PR